hw/mcu/nordic/src/ext/nrfx/templates/nrfx_glue.h

```diff
@@ -98,6 +105,27 @@ extern "C" {
  */
 #define NRFX_IRQ_DISABLE(irq_number)
 
+/**
+ * @brief Macro for setting a specific IRQ as pending.
+ *
+ * @param irq_number  IRQ number.
+ */
+#define NRFX_IRQ_PENDING_SET(irq_number)
```

Comment:
This and some other new macros should be updated(i.e. defined) on [mcu/nordic/include/nrfx_glue.h](https://github.com/apache/mynewt-core/blob/master/hw/mcu/nordic/include/nrfx_glue.h).